airflow/providers/pinecone/hooks/pinecone_hook.py

```diff
@@ -0,0 +1,122 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Hook for Pinecone."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+import pinecone
+
+from airflow.hooks.base import BaseHook
+
+if TYPE_CHECKING:
+    from pinecone.core.client.models import (
+        UpsertResponse,
+    )
```

Comment:
Nit, but did Black put this import into multiple lines? Or was there a comma at the end of the statement which forced Black's behavior?
```suggestion
    from pinecone.core.client.models import UpsertResponse
```

Reply:
Not, sure. I'll include this change and check if CI fails.